packages/jobs/lib/crons/autoIdleDemo.integration.test.ts

```diff
@@ -32,13 +33,38 @@ describe('Auto Idle Demo', async () => {
         });
 
         const connName = nanoid();
-        await configService.createProviderConfig({
+        const config = await configService.createProviderConfig({
             unique_key: DEMO_GITHUB_CONFIG_KEY,
             provider: 'github',
             environment_id: env.id,
             oauth_client_id: '',
             oauth_client_secret: ''
         });
+        await db.knex
+            .from<SyncConfig>('_nango_sync_configs')
+            .insert({
+                created_at: new Date(),
+                sync_name: DEMO_SYNC_NAME,
+                nango_config_id: config![0]!.id!,
+                file_location: '_LOCAL_FILE_',
+                version: '1',
+                models: ['GithubIssueDemo'],
+                active: true,
+                runs: 'every 5 minutes',
+                input: '',
+                model_schema: [],
+                environment_id: env.id,
+                deleted: false,
+                track_deletes: false,
+                type: SyncConfigType.SYNC,
+                auto_start: false,
+                attributes: {},
+                metadata: {},
+                pre_built: true,
+                is_public: false,
+                enabled: true
+            })
+            .returning('id');
```

Comment:
can we not use `sync.service.createSync` function?

Reply:
I need a sync_config not a sync there is no function for that in sync/config.service.ts